db/schema.rb

```diff
@@ -621,6 +621,20 @@
   add_index "secure_rooms_card_readers", ["product_id"], name: "index_secure_rooms_card_readers_on_product_id", using: :btree
   add_index "secure_rooms_card_readers", ["tablet_token"], name: "index_secure_rooms_card_readers_on_tablet_token", unique: true, using: :btree
 
+  create_table "secure_rooms_events", force: :cascade do |t|
+    t.integer  "card_reader_id", limit: 4
+    t.integer  "product_id",     limit: 4
+    t.integer  "user_id",        limit: 4
+    t.datetime "occurred_at"
```

Comment:
Since this timestamp is related to billing for use of the product, I'm less comfortable with depending on `created_at` to store it.